webviz_subsurface/plugins/_horizon_uncertainty_viewer.py

```diff
@@ -0,0 +1,837 @@
+        )
+        # pylint: disable=unused-variable, unused-argument
+        def display_output(n_clicks, column_list):
+            """ Renders wellpoints table from csv file """
+            wellpoints_df = self.df_well_target_points.update_wellpoints_df(column_list)
+            return html.Div(
+                [
+                    dash_table.DataTable(
+                        id=self.ids("well-points-table"),
+                        columns=[{"name": i, "id": i} for i in wellpoints_df.columns],
+                        data=wellpoints_df.to_dict("records"),
+                        sort_action="native",
+                        filter_action="native",
+                    ),
+                ]
+            )
+
+        @app.callback(
+            Output(self.ids("modal-table-settings"), "is_open"),
+            [
+                Input(self.ids("button-open-table-settings"), "n_clicks"),
+                Input(self.ids("button-close-table-settings"), "n_clicks"),
+            ],
+            [State(self.ids("modal-table-settings"), "is_open")],
+        )
+        def _toggle_modal_table_settings(n_open, n_close, is_open):
+            """ Open or close table settings modal button """
+            if n_open or n_close:
+                return not is_open
+            return is_open
+
+        @app.callback(
+            [
+                Output(self.ids("hidden-div-map-view"), "hidden"),
+                Output(self.ids("hidden-div-table-view"), "hidden"),
+            ],
+            [Input(self.ids("map-table-radioitems"), "value")],
+        )
+        def _toggle_left_flexbox_content(value):
+            """ Returns which left flexbox content is visible/hidden """
+            if value == "table-view":
+                switch = True, False
+            else:
+                switch = False, True
+            return switch
+
+        @app.callback(
+            Output(self.ids("uncertainty-table"), "data"),
+            [Input(self.ids("well-dropdown"), "value")],
+        )
+        def _render_uncertainty_table(wellfile):
+            if wellfile in self.wellfiles:
+                well = self.wells[wellfile]
+            else:
+                well = self.planned_wells[wellfile]
+            df = self.xsec.get_intersection_dataframe(well)
+            return df.to_dict("records")
+
+        @app.callback(
+            Output(self.ids("surface-picks-label"), "children"),
+            [Input(self.ids("well-dropdown"), "value")],
+        )
+        def _render_surface_picks_label(wellfile):
+            if wellfile in self.wellfiles:
+                wellname = self.wells[wellfile].wellname
+            else:
+                wellname = self.planned_wells[wellfile].wellname
+            return f"Surface picks for {wellname}"
+
+    def add_webvizstore(self):
+        return [(get_path, [{"path": fn}]) for fn in self.files]
+
+
+@webvizstore
+def get_path(path) -> Path:
+    return Path(path)
+
+
+def get_color(i):
```

Comment:
Do we have such a function already (with other plugins wanting different surface colors)?